core/option/options.go

```diff
@@ -612,6 +612,7 @@ func (opt OptMaxScan) String() string {
 }
 
 // OptMaxTime is for internal use.
+// This option is the same for CountDocuments and EstimatedDocumentCount.
```

Comment:
We don't need this comment line anymore since we've removed `EstimastedDocumentCountOptioner`.

Reply:
Done.